xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -727,30 +713,36 @@ bool CGUIMediaWindow::GetDirectory(const std::string &strDirectory, CFileItemLis
 // \brief Set window to a specific directory
 // \param strDirectory The directory to be displayed in list/thumb control
 // This function calls OnPrepareFileItems()
-bool CGUIMediaWindow::Update(const std::string &strDirectory, bool updateFilterPath /* = true */)
+bool CGUIMediaWindow::Update(const std::string &_arg, bool updateFilterPath /* = true */)
```

Comment:
-1 for using underscores to mark them as "throwaway" args and using `arg` as parameter name in this case.

We have tons of methods where we use const parameters only used to check something and process with a copy of that parameter. The same you do here and BTW you break the readability of the code because who knows what `_arg` is. To understand the code now, you need to go to the method docs and check the comment where you find "\param _arg The current directory path" .. that's really annoying.